cmd/mdatagen/documentation.tmpl

```diff
@@ -9,10 +9,22 @@ These are the metrics available for this scraper.
 | Name | Description | Unit | Type | Attributes |
 | ---- | ----------- | ---- | ---- | ---------- |
 {{- range $metricName, $metricInfo := .Metrics }}
-| {{ $metricName }} | {{ $metricInfo.Description }}{{ if $metricInfo.ExtendedDocumentation }} {{ $metricInfo.ExtendedDocumentation }}{{ end }} | {{ $metricInfo.Unit }} | {{ $metricInfo.Data.Type }}
+| {{ if and $.ExpGen $metricInfo.IsEnabled }}**{{ end }}{{ $metricName }}{{ if and $.ExpGen $metricInfo.IsEnabled }}**
```

Comment:
Confused about the need of ExpGen. In the old format all metrics are enabled correct? So why do you need this?

Reply:
They are enabled, but there is no way to reconfigure it, so I left it as is to not bring confusion

Reply:
Ok, tried that. All metrics from the old generators are highlighted now but with no info about optional metrics. Let me know how does it look now